### PR TITLE
Reformatted the delete, list, debug topics to match inspect.

### PR DESCRIPTION
**Pull request by stuclem**
Rearranged content in the topics on `vic-machine delete`, `ls`, `debug` in order to make them more descriptive than procedural. Consolidated content and removed the options reference topics to reduce duplication and clicks. This reorg reflects what I did in the `vic-machine inspect` updates in https://github.com/vmware/vic-product/pull/460

@anchal-agrawal these changes are entirely presentation-based, but you're always a very eager reviewer of of doc PRs!

### CODE REVIEW

File: docs/user_doc/vic_vsphere_admin/remove_vch.md [hunk 1]
```diff
@@ -2,44 +2,41 @@
 
 You delete virtual container hosts (VCHs) by using the `vic-machine delete` command.
 
-For descriptions of the options that `vic-machine delete` includes in addition to the [Common `vic-machine` Options](common_vic_options.md) , see [VCH Delete Options](delete_vch_options.md).
+The `vic-machine delete` includes one option in addition to the [Common `vic-machine` Options](common_vic_options.md) , `--force`.
```
Comment (anchal-agrawal) [inline docs/user_doc/vic_vsphere_admin/remove_vch.md]: Minor formatting issue (space before comma) - this should be
```
...(common_vic_options.md), `--force`.
```

File: docs/user_doc/vic_vsphere_admin/vch_shell_access.md [hunk 2]
```diff
@@ -4,33 +4,30 @@ You can use the `vic-machine debug` command to enable shell access to a virtual
...
 
     <pre>$ vic-machine-<i>operating_system</i> debug
      --target <i>vcenter_server_or_esxi_host_address</i>
      --user <i>vcenter_server_or_esxi_host_username</i>
      --password <i>vcenter_server_or_esxi_host_password</i>
      --id <i>vch_id</i>
      --thumbprint <i>certificate_thumbprint</i>
-     --rootpw '<i>new_p@ssword</i>' 
-</pre>
+     --rootpw '<i>new_p@ssword</i>' </pre>
```
Comment (anchal-agrawal) [inline docs/user_doc/vic_vsphere_admin/vch_shell_access.md]: This isn't rendering correctly - it's showing the raw tags in the markdown.

File: docs/user_doc/vic_vsphere_admin/remove_vch.md (hunk 1 above)
Comment (stuclem) [inline docs/user_doc/vic_vsphere_admin/remove_vch.md]: Fixed!

File: docs/user_doc/vic_vsphere_admin/vch_shell_access.md (hunk 2 above)
Comment (stuclem) [inline docs/user_doc/vic_vsphere_admin/vch_shell_access.md]: Again, line breaks do funny things to the rendering of code blocks in Github vs Gitbook HTML. Fixed!

**Comment by stuclem** [comment]
Thanks @anchal-agrawal !

**Pull request merged**